example/boa/tf2/first-tensorflow.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
```

Comment:
rename to hello.js? tensorflow might be duplicated with tf2.

Reply:
renamed

Reply:
I plan to add some boa demo for python and tensorflow, and have you some todo list？

Reply:
Python?

Reply:
Depending on your develop plan~